plugins/processors/noise/README.md

```diff
@@ -0,0 +1,37 @@
+# Noise Processor
+
+The *Noise* processor is used to add noise to numerical field values. For each field a noise is generated using Laplace distribution and added to the value.
+Using the _sensitivity_ and _epsilon_ parameter the effect of the noise can be modified. A bigger _epsilon_ (compared to a smaller _sensitivity_) value means that noise variation will be smaller and the the resulting value will be more close to the real value.
+
+### Configuration
+```toml
+[[processors.noise]]
+  sensitivity = 1.0
+  epsilon = 1.0
+  ignore_fields = []
+  ignore_measurements = []
+```
+
+### Example
+Add noise to each value the *Inputs.CPU*  plugin generates, except for the _uptime\_format_ field and all fields of the metrics _swap_, _disk_ and _net_:
+
+```toml
+[[inputs.cpu]]
+  percpu = true
+  totalcpu = true
+  collect_cpu_time = false
+  report_active = false
+
+[[processors.noise]]
+  sensitivity = 1.0
```

Comment:
Would it make sense to also let the user change the type of the noise e.g. `laplacian`, `gaussian`, `uniform`? This would open up the processor beyond differential privacy.

Reply:
Yes this makes sense. I will address the support for these other types in the next commit.

Reply:
Implemented with da8adb88d10e9a1d5d9d278971a37a2032dfd843